.github/workflows/scale-test-gce.yaml

```diff
@@ -36,8 +36,8 @@ env:
   go_version: 1.21.3
   # Adding k8s.local to the end makes kops happy-
   # has stricter DNS naming requirements.
-  cluster_name: ${{ github.repository_owner }}-${{ github.event.repository.name }}-${{ github.run_id }}-${{ github.run_attempt }}.k8s.local
-  cl2_config: ${{ github.workspace }}/.github/actions/scale-test-gce/cl2-config.yaml
+  test_name: node-throughput
+  cluster_name: ${{ github.run_id }}-${{ github.run_attempt }}.k8s.local
```

Comment:
Can you switch these two lines? I think the `cluster_name` should be directly underneath the comment about k8s.local.